inst/tinytest/helpers.R

```diff
@@ -2,9 +2,9 @@ library(tinytest)
 library(tinysnapshot)
 
 # Skip tests if not on Linux
-ON_LINUX = Sys.info()["sysname"] == "Linux"
-if (!ON_LINUX) exit_file("Linux snapshots")
-
-options("tinysnapshot_os" = "Linux")
+# ON_LINUX = Sys.info()["sysname"] == "Linux"
```

Comment:
That was unintentional. But it turns out the tests work on my local mac. I still reverted it back, but it might be worth looking into expanding the test suite eventually. Probably low priority.